packages/block-editor/src/components/rich-text/test/index.perf-test.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { measurePerformance } from 'reassure';
+import { fireEvent } from '@testing-library/react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+
+describe( 'RichText Performance', () => {
+	it( 'measures performance', async () => {
+		const onChange = jest.fn();
+
+		const changeTextOfRichText = ( richText, text ) => {
+			fireEvent( richText, 'focus' );
+			fireEvent( richText, 'onChange', {
+				nativeEvent: {
+					eventCount: 1,
+					target: undefined,
+					text,
+				},
+			} );
+		};
+
+		const scenario = async () => {
+			const richTextInput = <RichText />;
+			// Simulate user typing text.
+			fireEvent( richTextInput, 'focus' );
```

Comment:
This test fails currently, but that is expected to me as this line is improper to my understanding. This appears to set `richTextInput` to a separate copy of `RichText` than the one passed to `measurePerformance`. The test currently fails with the following error:

```
TypeError: Cannot read property 'parent' of undefined
```

My thought is that maybe this should reference `screen` passed to the `scenario` callback. 

```suggestion
		const scenario = async (screen) => {
			// Simulate user typing text.
			fireEvent( screen, 'focus' );
```